colorbleed/maya/__init__.py

```diff
@@ -112,13 +112,18 @@ def on_save(_):
     for node, new_id in lib.generate_ids(nodes):
         lib.set_id(node, new_id, overwrite=False)
 
+    lib.validate_fps()
+
 
 def on_open(_):
     """On scene open let's assume the containers have changed."""
 
     from avalon.vendor.Qt import QtWidgets
     from ..widgets import popup
 
+    # Ensure scene's FPS is set to project config
+    lib.set_project_fps()
```

Comment:
Don't set the FPS - it'll be too confusing for the artist that somehow the scene just magically changed. It should always just be a clear warning pop-up that states your currently set FPS and what it should be according to the project.